protocol/x/clob/abci.go

```diff
@@ -156,11 +164,26 @@ func PrepareCheckState(
 	)
 
 	// 3. Place all stateful order placements included in the last block on the memclob.
+	// Note telemetry is measured outside of the function call because `PlaceStatefulOrdersFromLastBlock`
+	// is called within `PlaceConditionalOrdersTriggeredInLastBlock`.
+	startPlaceLongTermOrders := time.Now()
```

Comment:
Avoided adding this within `PlaceStatefulOrdersFromLastBlock` since I thought the double counting would be confusing.